components/NavigationDrawer/src/MDCBottomDrawerViewController.m

```diff
@@ -52,13 +53,16 @@ - (void)commonMDCBottomDrawerViewControllerInit {
   _topCornersRadius[@(MDCBottomDrawerStateCollapsed)] = @(0);
   _maskLayer = [[MDCBottomDrawerHeaderMask alloc] initWithMaximumCornerRadius:0
                                                           minimumCornerRadius:0];
+  _maskApplied = NO;
```

Comment:
Nit: Please don't initialize instance variables to `@NO` or `nil`, since they are done automatically.

http://google.github.io/styleguide/objcguide.html#initialization